README.md

```diff
@@ -166,11 +166,15 @@ Yes, in fact - it has nothing to do with it. What it does is just linking static
 Sure you can! Try doing something like this in your package.json:
 ```json
 ...
-"scripts": {
-  "postinstall": "rnpm link"
+"dependencies": {
+  "rnpm": "^1.4.1"
 },
+"scripts": { 
+  "postinstall": "cd ../.. && rnpm link <name>"
+}
 ...
 ```
+Note that the `cd ../..` in the postinstall script is needed to go up to the working directory of the parent project that is to be linked with your plugin.
```

Comment:
Maybe I'm missing something. Usually users install our plugin from the root folder of their project, e.g npm install --save react-native-code-push. That will get saved in `./node_modules/react-native-code-push`, therefore, two directories deep from the parent project. The goal of this postinstall script is to link the plugin with the parent project, therefore doesn't it have to `cd` up two directories? I asked this in Discord but didnt really get a reply on this.